controllers/trillian/utils/trillian-db.go

```diff
@@ -63,17 +63,29 @@ func CreateTrillDb(instance *v1alpha1.Trillian, dpName string, sa string, opensh
 								ProbeHandler: core.ProbeHandler{
 									Exec: &core.ExecAction{
 										Command: []string{
-											"mysqladmin",
-											"ping",
-											"-h",
-											"localhost",
-											"-u",
-											"$(MYSQL_USER)",
-											"-p$(MYSQL_PASSWORD)",
+											"bash",
+											"-c",
+											"mariadb -u $MYSQL_USER -p${MYSQL_PASSWORD} -e \"SELECT 1;\"",
```

Comment:
Should this be 
```suggestion
											"mariadb -u ${MYSQL_USER} -p${MYSQL_PASSWORD} -e \"SELECT 1;\"",
```
if for nothing else than consistency?